lang/c++/CMakeLists.txt

```diff
@@ -264,3 +266,28 @@ if (NOT CMAKE_BUILD_TYPE)
       "Choose the type of build, options are: None Debug Release RelWithDebInfo MinSizeRel."
       FORCE)
 endif (NOT CMAKE_BUILD_TYPE)
+
+include(CMakePackageConfigHelpers)
+
+write_basic_package_version_file(
+    "${CMAKE_CURRENT_BINARY_DIR}/AvroConfigVersion.cmake"
+    VERSION ${AVRO_VERSION}
+    COMPATIBILITY SameMajorVersion)
+
+configure_package_config_file(
+    "${CMAKE_CURRENT_SOURCE_DIR}/cmake/AvroConfig.cmake.in"
+    "${CMAKE_CURRENT_BINARY_DIR}/AvroConfig.cmake"
+    INSTALL_DESTINATION lib/cmake/Avro
```

Comment:
I see an inconsistency here. The headers are installed in `include/avro` while cmake files are in `lib/cmake/Avro`. Maybe it should be `lib/cmake/avro` instead?

Reply:
I'm actually not sure which one is more preferable here. I'll probably want to hear from the committers to this repo regarding the naming.

Reply:
In my (poor) experience the convention is to use the one with the capital letter.

Reply:
Agreed. It is adopted by many libraries like Boost, Snappy and Arrow.